bin/hama jar ../../../../target/compare-hama-0.0.1-SNAPSHOT.jar comparison.pregelix.hama.PageRank /user/hadoop/data/webmap001x/part-all hama_PageRank 8 4
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: Starting Socket Reader #1 for port 61002
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server Responder: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO message.HamaMessageManagerImpl: BSPPeer address:sensorium-9.ics.uci.edu port:61002
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server handler 3 on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server listener on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server handler 1 on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server handler 4 on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server handler 0 on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 INFO ipc.Server: IPC Server handler 2 on 61002: starting
attempt_201404221826_0001_000005_0: 14/04/22 18:46:00 FATAL bsp.GroomServer: Error running child
attempt_201404221826_0001_000005_0: org.apache.hadoop.security.AccessControlException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RemoteException.instantiateException(RemoteException.java:95)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RemoteException.unwrapRemoteException(RemoteException.java:57)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3458)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient.create(DFSClient.java:870)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DistributedFileSystem.create(DistributedFileSystem.java:205)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:555)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:536)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:443)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:435)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.TextOutputFormat.getRecordWriter(TextOutputFormat.java:112)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.initializeIO(BSPPeerImpl.java:323)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.<init>(BSPPeerImpl.java:180)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.GroomServer$BSPPeerChild.main(GroomServer.java:1239)
attempt_201404221826_0001_000005_0: Caused by: org.apache.hadoop.ipc.RemoteException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:217)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:197)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.checkPermission(FSPermissionChecker.java:141)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkPermission(FSNamesystem.java:5758)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkAncestorAccess(FSNamesystem.java:5731)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFileInternal(FSNamesystem.java:1576)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFile(FSNamesystem.java:1527)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:710)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:689)
attempt_201404221826_0001_000005_0: 	at sun.reflect.GeneratedMethodAccessor38.invoke(Unknown Source)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:587)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1432)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1428)
attempt_201404221826_0001_000005_0: 	at java.security.AccessController.doPrivileged(Native Method)
attempt_201404221826_0001_000005_0: 	at javax.security.auth.Subject.doAs(Subject.java:415)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1190)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:1426)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Client.call(Client.java:1107)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
attempt_201404221826_0001_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
attempt_201404221826_0001_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3451)
attempt_201404221826_0001_000005_0: 	... 10 more
attempt_201404221826_0001_000005_0: org.apache.hadoop.security.AccessControlException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RemoteException.instantiateException(RemoteException.java:95)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RemoteException.unwrapRemoteException(RemoteException.java:57)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3458)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient.create(DFSClient.java:870)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DistributedFileSystem.create(DistributedFileSystem.java:205)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:555)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:536)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:443)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:435)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.TextOutputFormat.getRecordWriter(TextOutputFormat.java:112)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.initializeIO(BSPPeerImpl.java:323)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.<init>(BSPPeerImpl.java:180)
attempt_201404221826_0001_000005_0: 	at org.apache.hama.bsp.GroomServer$BSPPeerChild.main(GroomServer.java:1239)
attempt_201404221826_0001_000005_0: Caused by: org.apache.hadoop.ipc.RemoteException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:217)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:197)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.checkPermission(FSPermissionChecker.java:141)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkPermission(FSNamesystem.java:5758)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkAncestorAccess(FSNamesystem.java:5731)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFileInternal(FSNamesystem.java:1576)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFile(FSNamesystem.java:1527)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:710)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:689)
attempt_201404221826_0001_000005_0: 	at sun.reflect.GeneratedMethodAccessor38.invoke(Unknown Source)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:587)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1432)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1428)
attempt_201404221826_0001_000005_0: 	at java.security.AccessController.doPrivileged(Native Method)
attempt_201404221826_0001_000005_0: 	at javax.security.auth.Subject.doAs(Subject.java:415)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1190)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:1426)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.Client.call(Client.java:1107)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
attempt_201404221826_0001_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
attempt_201404221826_0001_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
attempt_201404221826_0001_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0001_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
attempt_201404221826_0001_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0001_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3451)
attempt_201404221826_0001_000005_0: 	... 10 more
bin/hama jar ../../../../target/compare-hama-0.0.1-SNAPSHOT.jar comparison.pregelix.hama.SSSP /user/hadoop/data/webmap001x/part-all hama_SSSP 8 33
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: Starting Socket Reader #1 for port 61001
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server Responder: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server listener on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server handler 0 on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server handler 2 on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server handler 1 on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server handler 3 on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO message.HamaMessageManagerImpl: BSPPeer address:sensorium-9.ics.uci.edu port:61001
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 INFO ipc.Server: IPC Server handler 4 on 61001: starting
attempt_201404221826_0003_000005_0: 14/04/22 18:57:59 FATAL bsp.GroomServer: Error running child
attempt_201404221826_0003_000005_0: org.apache.hadoop.security.AccessControlException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RemoteException.instantiateException(RemoteException.java:95)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RemoteException.unwrapRemoteException(RemoteException.java:57)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3458)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient.create(DFSClient.java:870)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DistributedFileSystem.create(DistributedFileSystem.java:205)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:555)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:536)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:443)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:435)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.TextOutputFormat.getRecordWriter(TextOutputFormat.java:112)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.initializeIO(BSPPeerImpl.java:323)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.<init>(BSPPeerImpl.java:180)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.GroomServer$BSPPeerChild.main(GroomServer.java:1239)
attempt_201404221826_0003_000005_0: Caused by: org.apache.hadoop.ipc.RemoteException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:217)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:197)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.checkPermission(FSPermissionChecker.java:141)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkPermission(FSNamesystem.java:5758)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkAncestorAccess(FSNamesystem.java:5731)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFileInternal(FSNamesystem.java:1576)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFile(FSNamesystem.java:1527)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:710)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:689)
attempt_201404221826_0003_000005_0: 	at sun.reflect.GeneratedMethodAccessor38.invoke(Unknown Source)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:587)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1432)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1428)
attempt_201404221826_0003_000005_0: 	at java.security.AccessController.doPrivileged(Native Method)
attempt_201404221826_0003_000005_0: 	at javax.security.auth.Subject.doAs(Subject.java:415)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1190)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:1426)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Client.call(Client.java:1107)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
attempt_201404221826_0003_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
attempt_201404221826_0003_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3451)
attempt_201404221826_0003_000005_0: 	... 10 more
attempt_201404221826_0003_000005_0: org.apache.hadoop.security.AccessControlException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RemoteException.instantiateException(RemoteException.java:95)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RemoteException.unwrapRemoteException(RemoteException.java:57)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3458)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient.create(DFSClient.java:870)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DistributedFileSystem.create(DistributedFileSystem.java:205)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:555)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:536)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:443)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.fs.FileSystem.create(FileSystem.java:435)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.TextOutputFormat.getRecordWriter(TextOutputFormat.java:112)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.initializeIO(BSPPeerImpl.java:323)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.BSPPeerImpl.<init>(BSPPeerImpl.java:180)
attempt_201404221826_0003_000005_0: 	at org.apache.hama.bsp.GroomServer$BSPPeerChild.main(GroomServer.java:1239)
attempt_201404221826_0003_000005_0: Caused by: org.apache.hadoop.ipc.RemoteException: org.apache.hadoop.security.AccessControlException: Permission denied: user=jianfenj, access=WRITE, inode="user":hadoop:supergroup:rwxr-xr-x
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:217)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:197)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.checkPermission(FSPermissionChecker.java:141)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkPermission(FSNamesystem.java:5758)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkAncestorAccess(FSNamesystem.java:5731)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFileInternal(FSNamesystem.java:1576)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.startFile(FSNamesystem.java:1527)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:710)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.server.namenode.NameNode.create(NameNode.java:689)
attempt_201404221826_0003_000005_0: 	at sun.reflect.GeneratedMethodAccessor38.invoke(Unknown Source)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:587)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1432)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:1428)
attempt_201404221826_0003_000005_0: 	at java.security.AccessController.doPrivileged(Native Method)
attempt_201404221826_0003_000005_0: 	at javax.security.auth.Subject.doAs(Subject.java:415)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1190)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:1426)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.Client.call(Client.java:1107)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
attempt_201404221826_0003_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
attempt_201404221826_0003_000005_0: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
attempt_201404221826_0003_000005_0: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
attempt_201404221826_0003_000005_0: 	at java.lang.reflect.Method.invoke(Method.java:606)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
attempt_201404221826_0003_000005_0: 	at com.sun.proxy.$Proxy2.create(Unknown Source)
attempt_201404221826_0003_000005_0: 	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream.<init>(DFSClient.java:3451)
attempt_201404221826_0003_000005_0: 	... 10 more
bin/hama jar ../../../../target/compare-hama-0.0.1-SNAPSHOT.jar comparison.pregelix.hama.TriagleCounting /user/hadoop/data/webmap001x/part-all hama_TriagleCounting 8
